### PR TITLE
Move "file does not exist" or "already exists" to debug log level

### PR DESCRIPTION
**Pull request by LuQQiu**
The return code already shows users the file does not exist or already exists or input path is invalid. There is no need to add duplicated error level logging which will make the log messy.

### CODE REVIEW

File: integration/fuse/src/main/java/alluxio/fuse/AlluxioFuseFileSystem.java [hunk 1]
```diff
@@ -429,6 +444,9 @@ public int open(String path, FuseFileInfo fi) {
         // Assuming I will never wrap around (2^64 open files are quite a lot anyway)
         mNextOpenFileId += 1;
       }
+    } catch (FileDoesNotExistException e) {
```
Comment (calvinjia) [inline integration/fuse/src/main/java/alluxio/fuse/AlluxioFuseFileSystem.java]: Invalid path can't be thrown here?

File: integration/fuse/src/main/java/alluxio/fuse/AlluxioFuseFileSystem.java [hunk 2]
```diff
@@ -357,8 +360,8 @@ public int getattr(String path, FileStat stat) {
       }
       stat.st_mode.set(mode);
       stat.st_nlink.set(1);
-    } catch (InvalidPathException | FileDoesNotExistException e) {
-      LOG.debug("Failed to get info of {}, path is invaid or does not exist", path, e);
+    } catch (FileDoesNotExistException e) {
```
Comment (calvinjia) [inline integration/fuse/src/main/java/alluxio/fuse/AlluxioFuseFileSystem.java]: Why did we remove the `InvalidPathException` from this catch?

File: integration/fuse/src/main/java/alluxio/fuse/AlluxioFuseFileSystem.java [hunk 3]
```diff
@@ -541,7 +544,7 @@ public int readdir(String path, Pointer buff, FuseFillDir filter,
         filter.apply(buff, file.getName(), null, 0);
       }
     } catch (FileDoesNotExistException | InvalidPathException e) {
-      LOG.debug("Failed to read directory {}, path is invalid or does not exist", path, e);
+      LOG.debug("Failed to read directory {}, path does not exist or is invalid", path, e);
```
Comment (calvinjia) [inline integration/fuse/src/main/java/alluxio/fuse/AlluxioFuseFileSystem.java]: Do we want to provide the full exception in the log? Currently we do it from places and not for others. I would prefer to not log the entire exception (ie. `LOG.debug("Failed to read directory {}, path does not exist or is invalid", path);`

File: integration/fuse/src/main/java/alluxio/fuse/AlluxioFuseFileSystem.java (hunk 1 above)
Comment (LuQQiu) [reply integration/fuse/src/main/java/alluxio/fuse/AlluxioFuseFileSystem.java]: Double checked in DefaultFileSystemMaster, Invalid path can be thrown here. Fixed, thanks

File: integration/fuse/src/main/java/alluxio/fuse/AlluxioFuseFileSystem.java (hunk 2 above)
Comment (LuQQiu) [reply integration/fuse/src/main/java/alluxio/fuse/AlluxioFuseFileSystem.java]: Added back, thanks

File: integration/fuse/src/main/java/alluxio/fuse/AlluxioFuseFileSystem.java (hunk 3 above)
Comment (LuQQiu) [reply integration/fuse/src/main/java/alluxio/fuse/AlluxioFuseFileSystem.java]: Removed the full exception in debug level logging, thanks

**Pull request merged**